### ISSUE TITLE
Installing fasttext in with conda

### ISSUE THREAD

**Issue by yoelatla** [conda/conda#6126]
Hi,
I am trying to install fasttext, and I having the following output:
what shall I do- 
λ conda install fasttext
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Current conda install:

               platform : win-64
          conda version : 4.3.27
       conda is private : False
      conda-env version : 4.3.27
    conda-build version : 3.0.25
         python version : 3.6.3.final.0
       requests version : 2.18.4
       root environment : C:\Users\Yoel\Anaconda3  (writable)
    default environment : C:\Users\Yoel\Anaconda3
       envs directories : C:\Users\Yoel\Anaconda3\envs
                          C:\Users\Yoel\AppData\Local\conda\conda\envs
                          C:\SPB_Data\.conda\envs
          package cache : C:\Users\Yoel\Anaconda3\pkgs
                          C:\Users\Yoel\AppData\Local\conda\conda\pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/fasttext/win-64
                          https://conda.anaconda.org/fasttext/noarch
                          https://repo.continuum.io/pkgs/main/win-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : C:\SPB_Data\.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.27 requests/2.18.4 CPython/3.6.3 Windows/8.1 Windows/6.3.9600
          administrator : True

`$ C:\Users\Yoel\Anaconda3\Scripts\conda-script.py install fasttext`

    Traceback (most recent call last):
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\exceptions.py", line 640, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\cli\main.py", line 140, in _main
        exit_code = args.func(args, p)
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\cli\main_install.py", line 80, in execute
        install(args, parser, 'install')
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\cli\install.py", line 160, in install
        linked_dists = install_linked(prefix)
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\core\linked_data.py", line 123, in linked
        return set(linked_data(prefix, ignore_channels=ignore_channels).keys())
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\core\linked_data.py", line 115, in linked_data
        load_linked_data(prefix, dist_name, ignore_channels=ignore_channels)
      File "C:\Users\Yoel\Anaconda3\lib\site-packages\conda\core\linked_data.py", line 36, in load_linked_data
        rec = json.load(fi)
      File "C:\Users\Yoel\Anaconda3\lib\json\__init__.py", line 299, in load
        parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
      File "C:\Users\Yoel\Anaconda3\lib\json\__init__.py", line 354, in loads
        return _default_decoder.decode(s)
      File "C:\Users\Yoel\Anaconda3\lib\json\decoder.py", line 339, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
      File "C:\Users\Yoel\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
        raise JSONDecodeError("Expecting value", s, err.value) from None
    json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by kalefranz** [comment]
Unfortunately you have a corrupt root environment.  To my knowledge, the only way for this to happen is to forcibly terminate conda while it is operating on the root environment.

Leaving this issue open, because we should add extra context to the error message to indicate which json file in the conda-meta directory is corrupt.

**Comment by goanpeca** [comment]
This has been fixed already on 4.5.x series as there is now

https://github.com/conda/conda/blob/4.5.x/conda/exceptions.py#L603

Which provides information like

```
CorruptedEnvironmentError: The target environment has been corrupted. Corrupted environments most commonly
occur when the conda process is force-terminated while in an unlink-link
transaction.
  environment location: /Users/gpena-castellanos/Continuum/conda/devenv/envs/test-broken-index
  corrupted file: /Users/gpena-castellanos/Continuum/conda/devenv/envs/test-broken-index/conda-meta/loghub-0.3.0-py36_1.json
```

The file `/Users/gpena-castellanos/Continuum/conda/devenv/envs/test-broken-index/conda-meta/loghub-0.3.0-py36_1.json` was manually "corrupted" to test this PR
